src/main/resources/log4j2.xml

```diff
@@ -1,21 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration>
+<Configuration status="WARN">
     <Properties>
-        <property name="spring.profiles.active">${env:SPRING_PROFILES_ACTIVE}</property>
+        <Property name="spring.profiles.active">${env:SPRING_PROFILES_ACTIVE:-local}</Property>
     </Properties>
 
     <Appenders>
-        <Routing name="routing">
+        <Routing name="Routing">
             <Routes pattern="${spring.profiles.active}">
+                <!-- Development-friendly pattern output -->
                 <Route key="local">
-                    <Console name="consolePattern" target="SYSTEM_OUT">
-                        <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n"/>
+                    <Console name="Console-Pattern" target="SYSTEM_OUT">
+                        <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %highlight{%-5level} %cyan{%c{1.}} - %msg%n"/>
                     </Console>
                 </Route>
+                
+                <!-- Production JSON output -->
                 <Route>
-                    <Console name="consoleJson" target="SYSTEM_OUT">
-                        <JSONLayout complete="false" compact="true" eventEol="true" properties="true" stacktraceAsString="true">
+                    <Console name="Console-JSON" target="SYSTEM_OUT">
+                        <JSONLayout complete="false" 
+                                  compact="true" 
+                                  eventEol="true" 
+                                  stacktraceAsString="true"
+                                  includeTimeMillis="true"
+                                  objectMessageAsJsonObject="true">
```

Comment:
Why these changes to the Production JSON output? DId you verify that the changes are compatible with Datadog?